src/os/bluestore/BlueStore.cc

```diff
@@ -3575,7 +3575,7 @@ void BlueStore::ExtentMap::reshard(
   auto cct = onode->c->store->cct; // used by dout
 
   dout(10) << __func__ << " 0x[" << std::hex << needs_reshard_begin << ","
-	   << needs_reshard_end << ")" << std::dec
+	   << needs_reshard_end << ") segment 0x" << segment_size << std::dec
```

Comment:
fmt::format()?

Reply:
I like fmt::format, but everything in ExtentMap::reshard is in "<<" style.